cdap-client-tests/src/test/java/co/cask/cdap/client/AdapterClientTestRun.java

```diff
@@ -56,58 +56,59 @@
   @Before
   public void setUp() throws Throwable {
     super.setUp();
-    clientConfig.setNamespace(Constants.DEFAULT_NAMESPACE_ID);
     adapterClient = new AdapterClient(clientConfig);
     appTemplateClient = new ApplicationTemplateClient(clientConfig);
   }
 
   @Test
   public void testAdapters() throws Exception {
-    List<AdapterDetail> initialList = adapterClient.list();
+    Id.Namespace namespace = Id.Namespace.DEFAULT;
```

Comment:
can we use ``Id.Namespace.DEFAULT``, instead of this extra variable?

Also, could you please create a jira to unify ``Id.Namespace.DEFAULT`` and ``Constants.DEFAULT_NAMESPACE_ID``? Shouldn't have the latter anymore.